apps/credential-service/lib/server.ts

```diff
@@ -14,13 +14,17 @@ interface AccessTokenAndRevocation {
   revokeToken: () => void;
 }
 
+interface RequestParameterHeaders extends IncomingHttpHeaders {
+  ng_repo_name: string;
+  ng_repo_owner: string;
+}
+
 /** Regex for matching authorization header uses. */
 const authorizationRegex = new RegExp(/Bearer (.*)/);
 /** The length of time in ms between heartbeat checks. */
-const heartBeatIntervalLength = 5000;
```

Comment:
What's the exact reasoning for the increase, to have it captured?

Reply:
Added into the comment, essentially we are accounting for local processing where we block the thread because JS is single threaded.